dashboard/frontend/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import Link from "next/link";
+import Image from "next/image";
+import { socket } from "../pages/_app";
+
+const Header = () => {
+    const handleCloseButton = () => {
+        if (confirm('Are you sure you want to exit?')) {
+            socket.emit("terminate");  
+        } 
+    };
+
+    return (
+        <nav className="flex items-center justify-between flex-wrap shadow px-4 py-4">
+            <Link href="/" passHref={true}>
+                <Image
```

Comment:
This is failing the build for the frontend `npm run build:client`. 

```markdown
Error: Image Optimization using Next.js' default loader is not compatible with `next export`.
  Possible solutions:
    - Use `next start` to run a server, which includes the Image Optimization API.
    - Use any provider which supports Image Optimization (like Vercel).
    - Configure a third-party loader in `next.config.js`.
    - Use the `loader` prop for `next/image`.
  Read more: https://nextjs.org/docs/messages/export-image-api
    at /Users/coditva/Code/elit-altum/newman-dashboard/node_modules/next/dist/export/index.js:14:785
    at async Span.traceAsyncFn (/Users/coditva/Code/elit-altum/newman-dashboard/node_modules/next/dist/telemetry/trace/trace.js:6:584)
```

Ref: https://nextjs.org/docs/basic-features/image-optimization#loader

Reply:
Fixed it!